dev/devicelab/README.md

```diff
@@ -4,6 +4,60 @@ This package contains the test framework and tests that run on physical devices.
 More generally the tests are referred to as "tasks" in the API, but since we
 primarily use it for testing, this document refers to them as "tests".
 
+You can see the continuous build results from the master branch at
+http://go/flutter-dashboard/build.html.
+
+# Running tests locally
+
+Do make sure your tests pass locally before deploying to the CI environment.
+Below is a handful of commands that run tests in a fashion very close to how the
+CI environment runs them. These commands are also useful when you need to
+reproduce a CI test failure locally.
+
+To run a test use option `-t` (`--task`):
```

Comment:
To avoid ambiguity, please use a comma to split these statements. ("To xxx, do yyy.") Do the same for the following commands.